data/sd/legislature/Rocky-D-Blare-4b8dc90a-ed50-4f4e-8d9a-dceb0344afd3.yml

```diff
@@ -1,21 +0,0 @@
-id: ocd-person/4b8dc90a-ed50-4f4e-8d9a-dceb0344afd3
-name: Rocky D. Blare
-given_name: Rocky
-family_name: Blare
-email: Rocky.Blare@sdlegislature.gov
-image: https://lawmakerdocuments.blob.core.usgovcloudapi.net/photos/blare_2019.jpg
-party:
-- name: Republican
-roles:
```

Comment:
@sroomf needs to be undeleted

Reply:
@sroomf but we also should add a previous role with this info:

- end_date: '2021-01-12'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:sd/government
  district: '21'